CLAUDE.md

```
@@ -636,26 +636,23 @@ Claude Code や AI エージェントが自律的に開発を進める際のワ
 
 #### 3. 検証 (Validation)
 
-コミット前に必ず以下をすべて実行します。
+**CRITICAL: タスク完了の必須条件**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)